src/app/(sok)/_utils/elasticSearchRequestBody.ts

```diff
@@ -533,16 +639,19 @@ function filterLocation(counties, municipals, countries, international = false)
                 must.push(mustObject);
             }
 
-            filter.nested.query.bool.should.push({
+            filter.nested.query.bool?.should?.push({
                 bool: {
                     must,
                 },
             });
         });
     }
 
-    const internationalObject = {
-        bool: {},
+    const internationalObject: BoolFilter = {
+        bool: {
+            must_not: [],
+            should: [],
```

Comment:
Hvis jeg tar vekk disse to endringene, så ser det ut til å filtrere på sted virker igjen.

Reply:
Litt lengre ned i denne funksjonen så har vi koden:
```
 if (
        Object.keys(internationalObject.bool).includes("must_not") ||
        Object.keys(internationalObject.bool).includes("should")
    ) {
        filter.nested.query.bool?.should?.push(internationalObject);
    }
```
og da tenker jeg siden at vi nå har definert  `must_not` og `should` til å være en array og ikke udefined, så legges det til en bool query som da gjør at filter ikke virker som forventet